.travis.yml

```diff
@@ -41,7 +40,7 @@ before_script:
 script:
     - mkdir -p build
     - cd build
-    - cmake ..
+    - cmake -D CMAKE_PREFIX_PATH:PATH=${DEPS_DIR}/protobuf/install ..
```

Comment:
Why didn't you write it like `CMAKE_PREFIX_PATH=${DEPS_DIR}/protobuf/install:${PATH}`? Does the current version do something different?